x-pack/plugins/fleet/server/services/package_policies/experimental_datastream_features.ts

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { ElasticsearchClient } from '@kbn/core-elasticsearch-server';
+import type { SavedObjectsClientContract } from '@kbn/core-saved-objects-api-server';
+
+import type { NewPackagePolicy, PackagePolicy } from '../../types';
+import { updateDatastreamExperimentalFeatures } from '../epm/packages/update';
+
+export async function handleExperimentalDatastreamFeatureOptIn({
```

Comment:
@kpollich Looking at this look like there is no way to disable the experiment once it has been enabled no?

Reply:
It will be disabled if `synthetic_source: false` comes through in `PackagePolicy.package` in the API request.